layouts/partials/shared/math-render-mathjax.html

```diff
@@ -0,0 +1,12 @@
+<script type="text/javascript">
+  MathJax = {
+    tex: {
+      inlineMath: [['\\(', '\\)']]
```

Comment:
MathJax uses `\(` and `\)` for inline math by default, so I don't think you need the config here.